### PR TITLE
Cloud release notes april 2024

### PR DESCRIPTION
**Pull request by jonat123**
## Description

Added new release notes for April 2024

_What did you add/update/change?_

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [x] New content
* [ ] Updates related to a new version
* [ ] Other

## Deadline (if relevant)

_When should the content be published?_

### CODE REVIEW

File: umbraco-cloud/release-notes/2024-04-releasenotes.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# April 2024
+
+## Key Takeaways
+
+* **New Project UI** - New intuitive and modern user interface for projects on Umbraco Cloud.
+
+
+## New Project UI
+
+From creating new environments, managing your team, or gaining insight into the performance of your project it has never been easier to use Umbraco Cloud.
```
Comment (eshanrnh) [inline umbraco-cloud/release-notes/2024-04-releasenotes.md:10]:
```suggestion
From creating new environments, managing your team, to gaining insight into the performance of your project it has never been easier to use Umbraco Cloud.
```

File: umbraco-cloud/release-notes/2024-04-releasenotes.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+With the new project UI and UX on Umbraco Cloud, we have made it simpler and more intuitive to manage your project on Umbraco Cloud.
```
Comment (eshanrnh) [inline umbraco-cloud/release-notes/2024-04-releasenotes.md:12]:
```suggestion
With the new UI and UX for projects on Umbraco Cloud, managing your project is now more straightforward and intuitive.
```

File: umbraco-cloud/release-notes/2024-04-releasenotes.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+When visiting your project, you'll notice that your project now has a sidebar, containing the different project settings.
+The environment overview have also gotten a new design as well, making it easier to manage your different environments.
```
Comment (eshanrnh) [inline umbraco-cloud/release-notes/2024-04-releasenotes.md:15]:
```suggestion
The environment overview has received a fresh design, making it easier to manage your different environments.
```

File: umbraco-cloud/release-notes/2024-04-releasenotes.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+With this design, we have created a new and  modern UI and UX making it even easier to manage your project.
```
Comment (eshanrnh) [inline umbraco-cloud/release-notes/2024-04-releasenotes.md:17]:
```suggestion
```

This line is already added above on line 12.

**Review by eshanrnh — CHANGES_REQUESTED** [review]
Thanks for the PR, @jonat123 🙌 I've added some comments to it.

**Comment by jonat123** [comment]
@eshanrnh thanks for your comments! 😄

I have added your suggestions!

**Review by eshanrnh — APPROVED** [review]
Looks good to me, @jonat123 💪 Merging it in.

**Pull request merged**